### PR TITLE
#6353: Profier signpost

### PR DESCRIPTION
**Pull request by mo-tenstorrent**
Add signposts to python profiling

### CODE REVIEW

File: docs/source/tt-metalium/tools/tracy_profiler.rst [hunk 1]
```diff
@@ -133,6 +133,34 @@ Similar to the pytest setup, calling the parent script with ``-p`` option will p
 **Note**, it is recommended to sandwich the function call between the enable and disable calls rather than having them as first and last calls in the function being profiled.
 This is because ``settrace`` and ``setprofile`` trigger on more relevant events when the setup is done previous to the function call.
 
+Signposts in Python Code
```
Comment (mo-tenstorrent) [inline docs/source/tt-metalium/tools/tracy_profiler.rst:136]: @nsmithtt Wanted to make sure this is inline with what you had in mind as well.

Comment (nsmithtt) [reply docs/source/tt-metalium/tools/tracy_profiler.rst:136]: This is exactly what I had in mind, thank you!

**Comment by mo-tenstorrent** [comment]
https://github.com/tenstorrent-metal/tt-metal/actions/runs/8559394059 psot commit passing

**Comment by mo-tenstorrent** [comment]
https://github.com/tenstorrent-metal/tt-metal/actions/runs/8572206440 profiler regression after rebase

**Pull request merged**